helpers/Delivery.php

```diff
@@ -0,0 +1,315 @@
+<?php
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2015 (original work) Open Assessment Technologies SA ;
+ *
+ */
+
+namespace oat\taoProctoring\helpers;
+
+use oat\oatbox\user\User;
+use oat\oatbox\service\ServiceManager;
+use \core_kernel_classes_Resource;
+use \common_session_SessionManager;
+use oat\taoProctoring\model\mock\WebServiceMock;
+
+class Delivery extends Proctoring
```

Comment:
I understand this is a mock, but all the methods present in this _helper_ are your app business so it should be formalized as services then.